app/code/Magento/InventorySales/Model/IsProductSalableCondition/BackOrderNotifyCustomerCondition.php

```diff
@@ -74,15 +74,20 @@ public function execute(string $sku, int $stockId, float $requestedQty): Product
             $backOrderQty = $requestedQty - $stockItemData[GetStockItemDataInterface::QUANTITY];
             if ($backOrderQty > 0) {
                 $errors = [
-                    $this->productSalabilityErrorFactory->create([
+                    $this->productSalabilityErrorFactory->create(
+                        [
                             'code' => 'back_order-not-enough',
                             'message' => __(
                                 'We don\'t have as many quantity as you requested, '
```

Comment:
It's clear, that you were not an author of this code, but still, it would make sense to clean up this concatenated string which would not work, while we are here :)

Reply:
Can you explain in a little more detail what it is you are after here.

Reply:
Please disregard my previous comment. It should be fine as is and I wasn't attentive enough.